packages/transaction-controller/src/types.ts

```diff
@@ -43,6 +44,7 @@ type TransactionMetaBase = {
   blockNumber?: string;
   deviceConfirmedOn?: WalletDevice;
   verifiedOnBlockchain?: boolean;
+  baseFeePerGas?: Hex;
```

Comment:
I've mentioned in the receipt PR that we can order these alphabetically so maybe a small conflict if that goes in first.